src/app/profile/addActivityModal/addActivityForm/AddActivityForm.test.tsx

```diff
@@ -1,11 +1,15 @@
 import userEvent from '@testing-library/user-event';
 
-import { render, screen, within } from 'tests';
+import { render, screen, within, fireEvent, waitFor } from 'tests';
 import { AddActivityForm } from './AddActivityForm';
 
+const mockSubmit = jest.fn((data) => {
```

Comment:
Lepiej jest robić instancje mockowanych funkcji jedną na scenariusz - wtedy mamy 100% pewności, że jego wywołanie które sprawdziliśmy dotyczy tego co poklikaliśmy w danym renderze.
Więc tutaj to wywalić i przenieść przed renderem w `fires callback...`